PRS1001-2000/PRS1016AbarraG.xml

```diff
@@ -52,7 +52,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <persName type="alt">ʾAbbā Sǝbbār</persName>
                <nationality type="Ethiopia post-Aksumite">Christian Ethiopia</nationality>
                <faith type="EOTC">EOTC</faith>
-               <occupation>daǧǧ</occupation>
+               <persName><roleName type="title">daǧǧāzmāč</roleName></persName>
```

Comment:
i would say occupation type military for dajjazmach actually...

Reply:
I don't know. I don't know the history of these titles well enough. I had searched for daǧǧāzmāč and found that it was most often encoded as title (but occupation type military is also used).

Reply:
it is both, it is a title and a military function.